modules/agar/src/main/ts/ephox/agar/api/TestStore.ts

```diff
@@ -0,0 +1,80 @@
+import { Assert } from '@ephox/bedrock-client';
+import { Optional } from '@ephox/katamari';
+
+import { Chain } from './Chain';
+import { Step } from './Step';
+
+export interface TestStore<T = string> {
+  add: (value: T) => void;
+  adder: (value: T) => () => void;
+  adderH: (value: T) => () => Optional<boolean>;
+  clear: () => void;
+  sClear: Step<any, any>;
+  cClear: Chain<any, any>;
+  sAssertEq: (label: string, expected: T[]) => Step<any, any>;
+  cAssertEq: (label: string, expected: T[]) => Chain<any, any>;
+  assertEq: (label: string, expected: T[]) => void;
+  assertSortedEq: (label: string, expected: T[], cmp?: (a: T, b: T) => number) => void;
+  sAssertSortedEq: (label: string, expected: T[], cmp?: (a: T, b: T) => number) => Step<any, any>;
+}
+
+export const TestStore = <T = string>(): TestStore<T> => {
+  let array: T[] = [];
+
+  const add = (value: T) => {
+    array.push(value);
+    // eslint-disable-next-line no-console
+    console.log('store.add', value, array);
```

Comment:
Not exactly something to fix here, but we don't include console logs in tests. It's another days discussion though given it used to do this.